.github/workflows/release-cli.yaml

```diff
@@ -166,6 +186,21 @@ jobs:
           gh release upload \
             --repo buildbuddy-io/bazel \
             --clobber \
-            "${{ needs.create-release.outputs.VERSION }}" \
+            "${{ needs.push-tag.outputs.version }}" \
             "${{ steps.build.outputs.BINARY }}" \
             "${{ steps.build.outputs.BINARY }}.sha256"
+
+  publish-release:
+    runs-on: ubuntu-20.04
+    needs: [push-tag, sync-plugins, build-artifacts]
+    steps:
+      - name: Checkout buildbuddy-io/bazel
+        uses: actions/checkout@v3
+        with:
+          repository: buildbuddy-io/bazel
+          path: bazel-fork
+          token: ${{ secrets.BUILDBUDDY_GITHUB_USER_TOKEN }}
```

Comment:
nit: rm this part?

Reply:
Sorry which part?

Reply:
I don't think we need to check out `buildbuddy-io/bazel` since we're just running `gh release edit` passing in the repo name, no?

Reply:
Oh nice - tested that and it worked!